### PR TITLE
add initial frontend

### PR DESCRIPTION
**Pull request by HumayraHanif**

### CODE REVIEW

**Review by wgcisotto — APPROVED** [review]

File: Frontend/src/App.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <th>Mouth</th>
+        <th>Accessory</th>
+        <th>Score</th>
+      </tr>
+    </thead>
+    <tbody>
+      {data.list.map(item => (
+              <tr key={item.id}>
+                <td>{item.id}</td>
+                <td>{item.metadata.Background}</td>
```
Comment (wgcisotto) [inline Frontend/src/App.js:52]: all the properties of metadata is in lowercase (update in the backend)

**Review by wgcisotto — CHANGES_REQUESTED** [review]
all the properties of metadata is in lowercase (update in the backend)

File: Frontend/src/App.js [hunk 2]
```diff
@@ -6,12 +6,12 @@ function App() {
...
     "id": 0,
     "metadata": {
-      "Background": "string",
-      "Left_Eye": "string",
-      "Face": "string",
-      "Right_Eye": "string",
-      "Mouth": "string",
-      "Accessory": "string",
+      "background": "string",
+      "left_Eye": "string",
```
Comment (web3allday) [inline Frontend/src/App.js:10]: should be left_eye right_eye everthing else fine

Comment (web3allday) [reply Frontend/src/App.js:10]:
```suggestion
      "left_eye": "string",
      "right_eye": "string",
```

File: Frontend/src/App.js [hunk 3]
```diff
@@ -6,12 +6,12 @@ function App() {
...
+      "face": "string",
+      "right_Eye": "string",
```
Comment (web3allday) [inline Frontend/src/App.js:12]: should be left_eye right_eye everthing else fine

Comment (web3allday) [reply Frontend/src/App.js:12]: change metadata to left_eye / right_eye

**Review by wgcisotto — APPROVED** [review]

**Pull request merged**